integration/dbt/scripts/dbt-ol

```diff
@@ -53,9 +54,10 @@ def main():
 
     events = processor.parse().events()
 
-    for event in events:
+    for event in tqdm(events, desc="Emitting OpenLineage events:"):
+        time.sleep(0.25)
         client.emit(event)
-    print(f"Emitted {len(events)} openlineage events")
+    print(f"Emitted {len(events)} OpenLineage events")
```

Comment:
can you print a message at the very beginning that mentions OpenLineage?
- the version (just like dbt does)
- that we're going to send events at the end.

Reply:
Done